### DISCUSSION TITLE
Resistance value of an aged cell

### DISCUSSION THREAD

**Discussion by MarcoZangrandi** [pybamm-team/PyBaMM#4492] [Q&A]
Hi, is there a variable with which I can identify the resistance value of an aged cell? I would like to do ageing cycles where I can see the decrease in capacitance due to ageing mechanisms and the relative increase in resistance. Thank you

**Comment by DrSOKane** [comment]
PyBaMM keeps track of most overpotentials. The file plot_voltage_components.py has a handy list of the variables you need:

```
        overpotentials = [
            "Battery negative particle concentration overpotential [V]",
            "Battery positive particle concentration overpotential [V]",
            "X-averaged battery negative reaction overpotential [V]",
            "X-averaged battery positive reaction overpotential [V]",
            "X-averaged battery concentration overpotential [V]",
            "X-averaged battery electrolyte ohmic losses [V]",
            "X-averaged battery negative solid phase ohmic losses [V]",
            "X-averaged battery positive solid phase ohmic losses [V]",
        ]
```

The electrolyte Ohmic losses and concentration overpotential will increases as the porosity decreases, so definitely keep track of those. The SEI also introduces an overpotential, which you can access as `"X-averaged negative electrode SEI film overpotential [V]"`.
